raw_data/preprocessed/africa/benin_french.txt

```diff
@@ -0,0 +1,303 @@
+
+CHAPITRE II - DE LA COMMISSION NATIONALE DU DÉVELOPPEMENT DURABLE
+
+(7) Il est institué un organisme dénommé "Commission nationale du Développement durable".
+
+(8) La Commission est composée de membres provenant du gouvernement et de la société civile. La Commission est dotée d'un secrétariat. La Commission peut faire appel à toute personne qu'elle jugera utile d'entendre ou de faire participer à ses travaux.
+
+(9) Chaque année, une dotation est inscrite au budget national pour le fonctionnement de la Commission.
+
+(10) Un décret pris en conseil des ministres précise le mandat, l'organisation, les modes d'élection ou de nomination des membres de la Commission, ainsi que son fonctionnement. 
+
+CHAPITRE III - DE L'AGENCE BÉNINOISE POUR L'ENVIRONNEMENT
+
+(11) Il est créé un établissement public, doté de la personnalité juridique et de l'autonomie financière, appelé "Agence béninoise pour l'Environnement" pour servir d'institution d'appui à la politique nationale en matière de protection de l'environnement.
+
+(12) L'Agence est chargée de la mise en oeuvre de la politique environnementale définie par le gouvernement dans le cadre du plan général de développement.
+
+(13) Les attributions de l'Agence, son organisation, les modalités de son fonctionnement et de son financement, ainsi que ses relations avec les institutions de l'État et des autres institutions sociales sont définies par un décret pris en conseil des ministres.
+
+(14) La contribution annuelle de l'État au fonctionnement de l'Agence est inscrite au budget de l'État. Elle lui est versée conformément à la législation en vigueur en matière de subvention aux établissements publics. 
+
+CHAPITRE IV - DE LA PROHIBITION GÉNÉRALE
+
+(15) Nul ne doit émettre, déposer, dégager, rejeter ou permettre l'émission, le dépôt, le dégagement, l'enfouissement ou le rejet dans l'environnement d'un contaminant au-delà de la quantité ou de la concentration prévue par les lois et règlements. Quiconque se rend coupable d'une pollution de l'environnement est tenu d'en réparer les conséquences conformément aux dispositions de la présente loi et des règlements y afférents et sans préjudice de l'application à son encontre des dispositions du code pénal.
+
+(16) Quiconque est responsable ou a connaissance de la présence accidentelle dans l'environnement d'un contaminant doit en aviser les autorités compétentes sous peine de poursuites pénales.
+
+(17) La divagation des animaux dans les agglomérations urbaines est interdite et punie conformément à la loi. 
+
+TITRE II - DE LA PROTECTION ET DE LA MISE EN VALEUR DES MILIEUX RÉCEPTEURS ET NATURELS
+
+CHAPITRE I - DU SOL ET DU SOUS-SOL
+
+(18) Au sens de la présente loi, est ‘‘sol'' tout terrain ou espace souterrain, même submergé d'eau ou couvert par une construction.
```

Comment:
The quotations here seem different from the normal " used, as seen in article 2. I.e. ‘‘sol'' instead of "sol". 
I don't see anything in the legislation to indicate they should be altered, not sure if this change is intentional. 
Note: These different quotations are used throughout this legislation in articles 45, 57, 66, 74, 87, 94 and 96.

Reply:
That's interesting. These marks certainly need to be replaced. I didn't notice the difference in Google Docs or in my terminal. I will update the articles that you listed. Thanks for pointing this out.